src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -57,6 +57,14 @@ public ResponseEntity<List<ModificationInfos>> getNetworkModifications(@Paramete
         return ResponseEntity.ok().body(networkModificationService.getNetworkModifications(groupUuid, onlyMetadata, errorOnGroupNotFound, onlyStashed));
     }
 
+    @GetMapping(value = "/groups/{groupUuid}/network-modifications-count", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Get a groups's modification count")
+    @ApiResponse(responseCode = "200", description = "Count of group's modifications")
+    public ResponseEntity<Integer> getNetworkModificationsCount(@Parameter(description = "Group UUID") @PathVariable("groupUuid") UUID groupUuid,
+                                                                @Parameter(description = "Stashed modifications") @RequestParam(name = "onlyStashed", required = false, defaultValue = "false") Boolean onlyStashed) {
```

Comment:
Rename `onlyStashed` by `stahed` because it's either not stahed or stahed but not all
Rename all int this PR

Reply:
Done